test/integration/start_stop_delete_test.go

```diff
@@ -103,13 +103,30 @@ func TestStartStop(t *testing.T) {
 					t.Fatalf("failed starting minikube -first start-. args %q: %v", rr.Command(), err)
 				}
 
+				// Soft Start this should not take longer than 10 seconds
+				softStartArgs := []string{"start", "-p", profile, "--alsologtostderr", "-v=3"}
+				rr, err = Run(t, exec.CommandContext(ctx, Target(), softStartArgs...))
+				if err != nil {
+					t.Fatalf("failed to soft start minikube. args %q: %v", rr.Command(), err)
+				}
+
+				// none driver does not invoke needs reset
+				// CNI ones will invoke a reset because the expected component coredns would not be running.
+				if !NoneDriver() && !strings.Contains(tc.name, "cni") {
+					// if this fails means, our soft start was a hard start.
+					softLog := "cluster does not need a reset"
+					if !strings.Contains(rr.Output(), softLog) {
+						t.Errorf("expected the soft start log outputs to include %q but got: %s", softLog, rr.Output())
+					}
+				}
+
 				if !strings.Contains(tc.name, "cni") {
 					testPodScheduling(ctx, t, profile)
 				}
 
 				rr, err = Run(t, exec.CommandContext(ctx, Target(), "stop", "-p", profile, "--alsologtostderr", "-v=3"))
 				if err != nil {
-					t.Errorf("failed stopping minikube - first stop-. args %q : %v", rr.Command(), err)
+					t.Errorf("failed stopping minikube - post soft start -. args %q : %v", rr.Command(), err)
```

Comment:
This text does not make sense to me. How about:

`failed to stop minikube after initial soft start`